gobcore/quality/config.py

```diff
@@ -35,7 +35,7 @@ class QA_CHECK:
 
     # Format requirements
     Format_N8 = {
-        "pattern": "^\d{8}$",
+        "pattern": r"^\d{8}$",
```

Comment:
Is hier een literal string nodig?
Maar voor reguliere expressies kan het geen kwaad om altijd gebruik te maken van literal strings. Maar in dat geval zou ik het voor alle strings doen, dus ook voor de overige wijzigingen in deze PR

Reply:
I changed all regex checks